.circleci/config.yml

```diff
@@ -70,30 +70,33 @@ jobs:
     <<: *test_settings
     parallelism: 4
     docker:
-      - image: python:2.7
+      - image: python:2.7-stretch
   py35:
     <<: *test_settings
     parallelism: 4
     docker:
-      - image: python:3.5
+      - image: python:3.5-stretch
 
   lint:
     docker:
-      - image: python:3.6
+      - image: python:3.6-stretch
     working_directory: ~/python_mozetl
     steps:
       - checkout
       - run: *early_return_for_skip_tests
       - restore_cache: *restore_cache_settings
       - run: *install_dependencies
+      - run:
+          name: install flake8 and black
+          command: pip install flake8 black
```

Comment:
This step should be unnecessary because they;re listed as dependencies in `tox.ini`

Reply:
Odd - I must've had an unrelated problem causing flake8 to not be found.  I've removed the step now.